### PR TITLE
Standalone component visualization

### PR DESCRIPTION
**Pull request by seicke**
Possibility to display only component visualization (without multi-column view)

URL parameters <code>aas</code> and <code>path</code> needed: <code>https://...:3000/componentvisualization?aas=...&path=...</code>

Bildschirmfoto 2024-05-13 um 06 59 33

@aaronzi: Please review

### CODE REVIEW

**Review by aaronzi — CHANGES_REQUESTED** [review]
Can you change `SubmodelList` in `aas-gui/Frontend/aas-web-gui/src/app.vue` in line 86 to `ComponentVisualization` so that the same thing works on mobile as well?

Otherwise, it looks good. Thank you for your contribution.

File: aas-gui/Frontend/aas-web-gui/src/components/ComponentVisualization.vue
Comment (aaronzi) [inline aas-gui/Frontend/aas-web-gui/src/components/ComponentVisualization.vue:1]: Is there a reason you changed 100svh to 100vh?

Comment (seicke) [reply aas-gui/Frontend/aas-web-gui/src/components/ComponentVisualization.vue:1]: By mistake, sorry

**Comment by seicke** [comment]
> Can you change `SubmodelList` in `aas-gui/Frontend/aas-web-gui/src/app.vue` in line 86 to `ComponentVisualization` so that the same thing works on mobile as well?
> 
> Otherwise, it looks good. Thank you for your contribution.

Wait, shouldn't it be:

```javascript
...
} else if (this.$route.name === 'ComponentVisualization') {
    if (!aasEndpoint && !submodelElementPath) {
        this.$router.push({ name: 'AASList' });
    } else if (aasEndpoint && !submodelElementPath) {
        this.$router.push({ name: 'SubmodelList', query: { aas: aasEndpoint } });
    }
}
...
```

**Comment by aaronzi** [comment]
Yes, you are right.

**Review by aaronzi — APPROVED** [review]
Thank you for the changes!

**Comment by FrankSchnicke** [comment]
Thanks a lot for your PR!

**Pull request merged**